diff --unified --recursive sofia-sip-1.12.11-orig/libsofia-sip-ua/su/sofia-sip/su_os_nw.h sofia-sip-1.12.11/libsofia-sip-ua/su/sofia-sip/su_os_nw.h
--- sofia-sip-1.12.11-orig/libsofia-sip-ua/su/sofia-sip/su_os_nw.h	2011-03-11 15:49:19.000000000 +0100
+++ sofia-sip-1.12.11/libsofia-sip-ua/su/sofia-sip/su_os_nw.h	2024-08-17 16:17:35.429206901 +0200
@@ -58,7 +58,7 @@
 #endif
 
 #if SU_HAVE_POLL
-#include <sys/poll.h>
+#include <poll.h>
 #endif
 
 SOFIA_BEGIN_DECLS
Només a sofia-sip-1.12.11/libsofia-sip-ua/su/sofia-sip: su_os_nw.h~
diff --unified --recursive sofia-sip-1.12.11-orig/libsofia-sip-ua/su/sofia-sip/su_wait.h sofia-sip-1.12.11/libsofia-sip-ua/su/sofia-sip/su_wait.h
--- sofia-sip-1.12.11-orig/libsofia-sip-ua/su/sofia-sip/su_wait.h	2011-03-11 15:49:19.000000000 +0100
+++ sofia-sip-1.12.11/libsofia-sip-ua/su/sofia-sip/su_wait.h	2024-08-17 16:17:40.893282362 +0200
@@ -47,7 +47,7 @@
 #endif
 
 #if SU_HAVE_POLL
-#include <sys/poll.h>
+#include <poll.h>
 #endif
 
 SOFIA_BEGIN_DECLS
Només a sofia-sip-1.12.11/libsofia-sip-ua/su/sofia-sip: su_wait.h~
